src/app/core/modal/modal.component.ts

```diff
@@ -36,6 +37,7 @@ export class ModalComponent {
         this.setScroll(false);
       }
       this.state = state;
+      ref.detectChanges();
```

Comment:
I think I added that specifically for the error-handling case.  A top-level error is caught without any zone changes... so the modal wouldn't appear with the stack trace until something else happened.

There may be a cleaner way to do this, only in the error-handling case.  Will look into it.